projects/common/xilinx/adi_fir_filter_bd.tcl

```diff
@@ -19,10 +19,19 @@ proc ad_add_decimation_filter {name filter_rate n_chan parallel_paths \
 
   create_bd_pin -dir I $name/aclk
   create_bd_pin -dir I $name/active
+  create_bd_pin -dir I $name/VCC
```

Comment:
Can't we create the VCC constant inside of the subsystem. It is strange to have a 'constant' on the interface but it can be connected to something else.

Reply:
Fixed. Thank you!